src/components/v2v3/V2V3Project/ProjectSafeDashboard/juiceboxTransactions/reconfigureFundingCyclesOf/ReconfigurationRichPreview.tsx

```diff
@@ -0,0 +1,101 @@
+import { t, Trans } from '@lingui/macro'
+import { Space } from 'antd'
+import { MinimalCollapse } from 'components/MinimalCollapse'
+import SplitList from 'components/v2v3/shared/SplitList'
+import FundingCycleDetails from 'components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails'
+import { ThemeContext } from 'contexts/themeContext'
+import { V2V3ContractsContext } from 'contexts/v2v3/V2V3ContractsContext'
+import { V2V3ProjectContext } from 'contexts/v2v3/V2V3ProjectContext'
+import { useContext } from 'react'
+import { formatOutgoingSplits } from 'utils/splits'
+import { formatReservedRate, MAX_DISTRIBUTION_LIMIT } from 'utils/v2v3/math'
+import { SafeTransactionComponentProps } from '../../SafeTransaction'
+
+export function ReconfigureRichPreview({
+  transaction,
+}: SafeTransactionComponentProps) {
+  const { contracts } = useContext(V2V3ContractsContext)
+  const { projectOwnerAddress } = useContext(V2V3ProjectContext)
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+
+  if (!contracts) return null
+
+  const decodedData = contracts['JBController'].interface.parseTransaction({
+    data: transaction.data ?? '',
+  }).args
+
+  const distributionLimitCurrency =
+    decodedData._fundAccessConstraints[0].distributionLimitCurrency
+  const distributionLimit =
+    decodedData._fundAccessConstraints[0].distributionLimit
+  const reservedRate = decodedData._metadata.reservedRate
+  const payoutSplits = decodedData._groupedSplits[0].splits
+  const reservedTokensSplits = decodedData._groupedSplits[1].splits
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column' }}>
+      <div style={{ margin: '1rem 3rem 0' }}>
+        {decodedData._memo.length ? (
+          <i>{decodedData._memo}</i>
+        ) : (
+          <i>
+            <Trans>
+              This transaction reconfigures the project's funding cycle.
+            </Trans>
+          </i>
+        )}
+      </div>
+      <Space
+        size={'middle'}
+        style={{ margin: '1rem 2rem' }}
+        direction={'vertical'}
+        onClick={e => e.stopPropagation()}
+      >
+        <MinimalCollapse header={t`Funding cycle details`} light>
+          <FundingCycleDetails
```

Comment:
@johnnyd-eth this is absolutely TIGHT! Great job. Just wanna check, is everything displayed here the same as what's displayed in the Review and Deploy data for create/reconfigure? It probably is, just checkin

Reply:
And also worth noting that: I think we're gonna be able to implement the redesign of Review and Deploy, and use the same component here, create flow and reconfigure. Fine for now though

Reply:
<img width="743" alt="Screen Shot 2022-09-28 at 5 08 39 pm" src="https://user-images.githubusercontent.com/96150256/192712035-855b8738-8a8d-4089-971f-63c743870374.png">

<img width="1025" alt="Screen Shot 2022-09-28 at 5 10 21 pm" src="https://user-images.githubusercontent.com/96150256/192712085-9821ab2e-236d-4582-8fad-d6a26adbe46a.png">

Yep it checks out